client/extensions/wp-super-cache/wrap-settings-form.jsx

```diff
@@ -50,16 +93,22 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 				handleToggle: this.handleToggle,
 			};
 
-			return <SettingsForm { ...this.props } { ...utils } />;
+			return (
+				<div>
+					<QuerySettings siteId={ this.props.siteId } />
```

Comment:
I still don't understand why you're rendering a null component here? What is the point of this?